ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -25,8 +25,11 @@ def __init__( self, *args, **kwargs ):
     AgentModule.__init__( self, *args, **kwargs )
     self.diracAdmin = None
     self.default_value = None
-    self.cacheFile = None
-    self.cacheFile = os.getenv('DIRAC') + 'work/ResourceStatus/cache.json'
+
+    if os.getenv('DIRAC'):
```

Comment:
```os.environ``` is a dict, so no reason for it to behave like string :-)

Just to note, however, there is a trick with tuples:
```
In [14]: t1 = ( 'something', 'else')

In [15]: t2 = ( 'something' )

In [16]: 'some' in t1
Out[16]: False

In [17]: 'some' in t2
Out[17]: True
```

If you have a single element in your tuple, without the ```,``` in it in the end, it is iterpreted as the type inside this tuple, and then you can fall in the pitfal you mention